go-controller/pkg/ovn/secondary_layer3_network_controller.go

```diff
@@ -148,6 +190,16 @@ func (h *secondaryLayer3NetworkControllerEventHandler) DeleteResource(obj, cache
 		}
 		return h.oc.deleteNodeEvent(node)
 
+	case factory.PodType:
```

Comment:
Why not just put this in DeleteSecondaryNetworkResourceCommon ?

Reply:
That's because isPodScheduledinLocalZone() is not defined in BaseSecondaryNetworkController struct.
That's because we don't support interconnect for layer2 and localnet networks yet.